main.go

```diff
@@ -248,16 +115,21 @@ func logWithTimestamp(coloringFunc ColoringFunc, format string, v ...interface{}
 }
 
 func main() {
-	configs := createConfigsModelFromEnvs()
-
-	fmt.Println()
-	configs.print()
-
-	if err := configs.validate(); err != nil {
+	var cfg configs
+	if err := stepconf.Parse(&cfg); err != nil {
 		fail("Issue with input: %s", err)
 	}
 
-	log.SetEnableDebugLog(configs.VerboseLog == "yes")
+	if cfg.CustomExportOptionsPlistContent != "" {
```

Comment:
pls also restore the `auto-detect` export method warning: https://github.com/bitrise-steplib/steps-xcode-archive/pull/151/files#diff-7ddfb3e035b42cd70649cc33393fe32cL114